uwblueprint/abtc
# Create serviceRequest implementation and interface

## Notion ticket link

[Create ServiceRequest implementation and interface](https://www.notion.so/uwblueprintexecs/Create-ServiceRequest-implementation-and-interface-83dbd95372e84a1b90ccb204f865530a?pvs=4)



## Implementation description
* Create skeleton for serviceRequest implementation and interface using Prisma



## Steps to test
1. Ensure build is successful
2. Check that Prisma syntax and definition is correct



## What should reviewers focus on?
* Correct syntax
* Successful build


## Checklist
